tests/e2e/performance/test_pvc_snapshot_performance.py

```diff
@@ -482,7 +482,10 @@ def test_pvc_snapshot_performance_multiple_files(
             log.info(f"Taking snapshot of the PVC {pvc_name}")
             log.info(f"Snapshot name : {snap_name}")
             creation_time = self.measure_create_snapshot_time(
-                pvc_name=pvc_name, snap_name=snap_name, interface=interface
+                pvc_name=pvc_name,
+                snap_name=snap_name,
+                namespace=BMO_NAME,
```

Comment:
There is one more testcase needs this change right ?

Reply:
One more test case is already merged as part of PR4861